cra-config.yaml

```diff
@@ -9,3 +9,4 @@ CRA_TARGETS:
       TF_VAR_instance_name: "mock"
       TF_VAR_environment_crn: "test-env-crn"
       TF_VAR_provider_visibility: "public"
+      TF_VAR_enable_cors: "true"
```

Comment:
I would not explicitly set this so we do a CRA scan against the default values of the DA